spinalcordtoolbox/utils.py

```diff
@@ -3,9 +3,8 @@
 # Collection of useful functions
 
 
-import io
-import os
-import re
+import io, sys, os
```

Comment:
doesn't it go against [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)?

Reply:
Yes indeed. I actually disabled the PEP8 on my editor due to too many warnings :sweat_smile: . 

If we want to be really consistent we could run the PEP8 formatter on the code (https://pypi.org/project/autopep8/). Developers could run it as a pre-commit hook (or manually) prior to pushing code.